lib/adapter.js

```diff
@@ -112,7 +112,10 @@ class MochaAdapter {
         if (params.err) {
             message.err = {
                 message: params.err.message,
-                stack: params.err.stack
+                stack: params.err.stack,
+                type: params.err.type || params.err.name,
+                expected: params.err.expected,
+                actual: params.err.actual
```

Comment:
isn't `actual` and `expected` a chai specific error property? What happens if someone doesn't use chai?